webpack.common.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
   entry: {
     digitalstrategie_berlin: [
       './digitalstrategie/assets/berlin_css/berlin_marketing.css',
-      './digitalstrategie/assets/berlin_css/shariff.min.css',
       './digitalstrategie/assets/berlin_css/fontawesome.all.min.css'
```

Comment:
@goapunk I tried adding the problematic font files locally also however it still triggers the `downloadable font: rejected by sanitizer (font-family: "Font Awesome 5 Brands" style:normal weight:400 stretch:100 src index:1) source: http://localhost:8006/static/fonts/fa-brands-400.woff2 ` error and it's always the same extensions so I'm now thinking it is the CSP or corrupted files? chrome gives better errors, it also has 
`The source list for the Content Security Policy directive 'style-src-elem' contains an invalid source: 'https://www.berlin.dehttp://berlin.de'. It will be ignored.`
maybe we can check together?

Reply:
Added extra commit to just load it via tag as it's not needed locally really, right?